proto/chart/v1/config.proto

```diff
@@ -11,3 +12,28 @@ message UpdateChartConfigs {
   // list of config hashes missing from cloud and requested from the agent
   repeated string config_hashes = 3;
 }
+
+message ChartConfigsUpdated {
+  repeated ChartConfig configs = 1;
+}
+
+message ChartConfig {
```

Comment:
This is probably a cloud-internal issue, as this functionality is inside the after the fat `ChartConfigUpdate` message is received.
According to our original sequence diagrams [here](https://netdata.atlassian.net/wiki/spaces/AR/pages/801439957/Chart-Message-Flow-NG#Use-Cases-(WIP)), the name for the individual ChartConfig messages was ChartConfigUpdate, same as the "fat" one just in singular for the Config(s).
So, to keep this uniform I will proceed to update this accordingly.
cc: @papazach 
```suggestion
message ChartConfigUpdated {
```

Reply:
Another choice could be to keep this as a chartConfig internal representation and use a different wrapper for the individual ChartConfigUpdated message.

Reply:
Cool not a problem at all Chris, just a minor naming change.

Reply:
Feel free to push the commit because I think that the suggested change here is not enough

Reply:
Thanks for the comment! Doing the proper full changes now 🚀